cg-spec/editors_draft.html

```diff
@@ -415,12 +415,125 @@ <h2>Concepts and Abstract Syntax</h2>
+        collectively known as <a>RDF Terms</a> as defined in [[RDF11-CONCEPTS]].
+      </p>
+      <p>
+        RDF-star introduces <a>quoted triple</a>, which is a new kind of RDF term.
+        A <em>quoted triple</em> is a triple used as the subject or object of another triple. Quoted
+        triples can also be called "embedded triples".
+      </p>
+
+      <p>
+        in RDF 1.1, an <a>asserted triple</a> is an element of the set of triples that
+        make up an <a>RDF graph</a>. RDF-star does not change this except that an RDF-star
+        triple can contain quoted triples. A triple can be used as an <em>asserted
+        triple</em>, a <em>quoted triple</em>, or both, in a given graph.
+      </p>
+
+      <p>
+        A quoted triple is written in Turtle-star and related syntaxes using delimiters `<<` and `>>`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << :a :name "Alice" >>
+        -->
+      </pre>  
+      <p>
+        The definition of quoted triple is recursive. A quoted triple can itself have a
+        subject or object component which is another quoted triple. Cycles of quoted
+        triples can not be created.  In this next example, there is a quoted triple with
+        property `:reportedBy` and whose subject is another quoted triple `:a :name
+        "Alice"`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << << :a :name "Alice" >> :reportedBy :charlie >>
+        -->
+      </pre> 
+    </section>
+
+    <section id="rdf-star-graph-examples">
+      <h2>RDF-star Graph Examples</h2>
+        
+      <p>
+        In the example below, there is a quoted triple, `:a :name "Alice"`, which is used
+        as the subject for an asserted triple:
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            PREFIX : <http://www.example.org/> 
+
+            :bob :name "Bob" .
+            << :a :name "Alice" >> :statedBy :bob .
+              -->
+      </pre> 
+      <p>
+        The graph of this example has two asserted triples:
+      </p>
+      <ol>
+        <li>`:bob :name "Bob" .`</li>
+        <li>`&lt;&lt; :a :name "Alice" &gt;&gt; :statedBy :bob .`</li>
+      </ol>
+      <p>
+        This graph does not have an asserted triple, `:a :name "Alice"`, because
+        that triple is not an element of the set of triples.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            PREFIX : <http://www.example.org/> 
+
+            :bob :name "Bob" .
+            :a :name "Alice" .
+            << :a :name "Alice" >> :statedBy :bob .
+              -->
+      </pre> 
+    </section>
+
+    <section id="annotation-syntax">
+      <h2>Annotation Syntax</h2>
```

Comment:
I'm not a big fan of this section here. We are in the "abstract syntax" section, and presenting the annotation syntax there, even while mentioning "this is not part of the abstract syntax", is confusing IMO.

The [Overview](https://w3c.github.io/rdf-star/cg-spec/editors_draft.html#overview) section already has a piece about the annotation syntax, and I think it is a better place for it. The only thing that the Overview does not show is that annotations can contain multiple triples, which could easily be fixed in [Example 2](https://w3c.github.io/rdf-star/cg-spec/editors_draft.html#example-annotation) if deemed necessary.

Reply:
Alternatively, it could go in the "Concrete Syntaxes" section. It might be too much detail for the Overview section.

Reply:
The overview section is already 50% about annotation syntax (and the example is bigger than the quoted triple one). More space on annotation syntax is skewing too much.  The overview could focus on "quoted triple", that being the important feature of RDF-star.

As @gkellogg suggests, moving it to the concrete syntax section, either as 3.1 and shift the rest, or the text goines into 3.Intro.

[Overview](https://w3c.github.io/rdf-star/cg-spec/editors_draft.html#overview) :
> This pattern is quite common

I'm not sure that it will be common, or more likely, in some uses it's all quoted (quoting triples elsewhere - provence) and some uses, it's all annotated.

Reply:
Annotation Syntax section moved with easy-to-revert commit 51bcebb.